src/screens/Home/Home.js

```diff
@@ -1,25 +1,73 @@
 import { useTheme } from '@react-navigation/native';
 import React from 'react';
-import { Text, View } from 'react-native';
-import { Config } from 'react-native-config';
+import { Text, View,ActivityIndicator, FlatList,Image, TouchableHighlight} from 'react-native';
```

Comment:
You are clearly missing a linter configuration, as we previously spoke. This file has a few `eslint` issues that can be fixed automatically once you set up your linter properly and enable the Visual Studio `eslint` extension

Reply:
ok!